### DISCUSSION TITLE
The relative time of commit is wrong

### DISCUSSION THREAD

**Discussion by sapjax** [community/community#106381] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="262" alt="image" src="https://github.com/community/community/assets/990255/fe56628c-25cb-448f-a39f-96792a637c5b">

like in the screenshot, it's not `2 years ago`, it's really `1 years ago`.

**Comment by gil-son** [comment]
Hi,

The time of a commit start when the commit is registered

For example:

If a guy make a commit in his computer yesterday and 3 days later send the updates (push) to Github

The Github will registry the duration of commit 3 day and not the currently day

**Reply by sapjax** [reply]
The time `Nov 11, 2022` in the screenshot is the commit time, not the push time.
You can't simply calculate the relative time by `2024 - 2022 = 2`, that's a bug.
